workflow-controller/controller-service/src/main/java/io/hyscale/controller/validator/impl/RegistryValidator.java

```diff
@@ -64,9 +66,12 @@
 	public boolean validate(WorkflowContext context) throws HyscaleException {
 	    logger.debug("Starting registry validation");
 	    Image image = context.getServiceSpec().get(HyscaleSpecFields.image, Image.class);
-	    
 	    String registry = image.getRegistry();
-	    
+	    ImageRegistry imageRegistry=null;
+		if (registry == null) {
```

Comment:
This will give false positive i.e. even if default registry credentials are not available, it will return true.

Reply:
Do we need this check? Calling RegistryManager manager with whatever value we get should be fine I think.